bash_it.sh

```diff
@@ -155,5 +166,17 @@ if ! command -v reload &> /dev/null && [ -n "$BASH_IT_RELOAD_LEGACY" ]; then
 	esac
 fi
 
+# Now that everything is loaded and configured, load the previewer
+BASH_IT_LOG_PREFIX='lib: preview: '
+# and the associated preview script
+# shellcheck source=./lib/preview.bash
+source "${BASH_IT}/lib/preview.bash"
+
 # Disable trap DEBUG on subshells - https://github.com/Bash-it/bash-it/pull/1040
 set +T
+
+# Finally, load preexec directly from vendor.
+# This has to be last because it wants to have full control over DEBUG trap and PROMPT_COMMAND
+BASH_IT_LOG_PREFIX='vendor: bash-preexec: '
+# shellcheck source=./vendor/github.com/rcaloras/bash-preexec/bash-preexec.sh
```

Comment:
also, I am not sure we need to specify the source here, as its an external library that we vendor, we should not fix the code here, but rather help upstream